app/services/asset_manager/asset_updater.rb

```diff
@@ -1,21 +1,24 @@
-class AssetManagerUpdateAssetWorker
-  include AssetManagerWorkerHelper
+class AssetManager::AssetUpdater
+  include AssetManager::ServiceHelper
 
-  class AssetManagerAssetDeleted < StandardError
+  class AssetAlreadyDeleted < StandardError
     def initialize(attachment_data_id, legacy_url_path)
-      message = "Asset corresponding to AttachmentData ID #{attachment_data_id} and legacy URL path #{legacy_url_path} expected not to be deleted in Asset Manager"
-      super(message)
+      super("Asset corresponding to AttachmentData ID #{attachment_data_id} and legacy URL path #{legacy_url_path} expected not to be deleted in Asset Manager")
```

Comment:
The message in `AssetManager::ServiceHelper::AssetNotFound` quotes the legacy URL path.  Should this message do the same?

Reply:
Yep, agreed! Done!